exporters/committime/app.py

```diff
@@ -1,254 +1,51 @@
 #!/usr/bin/python3
-import json
-import logging
+from collector_bitbucket import BitbucketCommitCollector
+from collector_gitlab import GitLabCommitCollector
+from collector_github import GitHubCommitCollector
 import os
 import pelorus
-import requests
-import re
 import time
-from datetime import datetime
-from jsonpath_ng import parse
 from kubernetes import client
-
 from openshift.dynamic import DynamicClient
 from prometheus_client import start_http_server
-from prometheus_client.core import GaugeMetricFamily, REGISTRY
+from prometheus_client.core import REGISTRY
 
-REQUIRED_CONFIG = ['GITHUB_USER', 'GITHUB_TOKEN']
+REQUIRED_CONFIG = ['GIT_USER', 'GIT_TOKEN']
 
 pelorus.load_kube_config()
 k8s_config = client.Configuration()
 k8s_client = client.api_client.ApiClient(configuration=k8s_config)
 dyn_client = DynamicClient(k8s_client)
+default_git = "github"
```

Comment:
we currently have all default values declared as constants in pelorus.py. Unless you have some argument, let's continue that pattern.